packages/workers-shared/asset-server-worker/src/index.ts

```diff
@@ -21,14 +20,13 @@ export default {
 		const url = new URL(request.url);
 		const { pathname } = url;
 
-		const isLocalDevContext = new Uint8Array(ASSETS_MANIFEST).at(0) === 1;
-		const assetsManifest = isLocalDevContext
-			? new NoopAssetsManifest()
-			: new AssetsManifest(ASSETS_MANIFEST);
-		const assetEntry = await assetsManifest.get(pathname);
-
-		const content = await ASSETS_KV_NAMESPACE.get(assetEntry);
+		const assetsManifest = new AssetsManifest(ASSETS_MANIFEST);
```

Comment:
Should `env.ASSETS_MANIFEST` ever be undefined? In the const variable declaration above it is defaulting to `new ArrayBuffer(0)` but I think that is outdated code, since we should "always" be providing a manifest now, right?